examples/tts/conf/fastpitch_align_v1.05.yaml

```diff
@@ -210,9 +210,9 @@ model:
     n_layers: 2
 
   optim:
-    name: lamb
+    name: adamw
     lr: 1e-1
-    betas: [0.9, 0.98]
```

Comment:
nit: could you please advise me any resource showing 0.999 is better than 0.98? sorry, I lack the experience of adamW.